diesel/src/backend.rs

```diff
@@ -345,6 +345,13 @@ pub(crate) mod sql_dialect {
         /// `QueryFragment` implementations
         pub trait SupportsOnConflictClause {}
 
+        /// A marker trait indicating if a `ON CONFLICT (...) DO UPDATE ... [WHERE ...]` clause is supported or not
+        pub trait SupportsOnConflictClauseWhere {}
+
+        /// A marker trait indicating whether the on conflict clause implementation
+        /// is mostly like postgresql
+        pub trait PgLikeOnConflictClause: SupportsOnConflictClause {}
```

Comment:
Does this syntax with the `:` character mean that `PgLikeOnConflictClause` is the same as `SupportsOnConflictClause`? Like a marker trait or something?